buildSrc/src/main/kotlin/pklFatJar.gradle.kts

```diff
@@ -94,6 +93,8 @@ tasks.shadowJar {
 
   configurations = listOf(project.configurations.runtimeClasspath.get())
 
+  addMultiReleaseAttribute = true
```

Comment:
This bumps the shadow plugin because Truffle failed to initialize with error like "multi-release lost" or something like that.

The `addMultiReleaseAttribute` is a new option available in the `org.gradleup` version of the plugin.